### DISCUSSION TITLE
Series Slight Improvement & Selectors Design Idea

### DISCUSSION THREAD

**Discussion by Perseusdehond** [jellyfin/jellyfin-vue#509] [UI & UX]
I am putting this idea, first shown in Matrix, also in here so it is easier to look at it for reference ;) This is an redesign of a redesign in a way haha. Now the concept is way more clean and easy to understand with updated explanation and design image.

Okay, so I heard what was mentioned about how you should select some tracks/subtitles for episodes on the series item page. That's why I thought about it and especially the error that would occur when some tracks aren't well defined. I'm not really a fan of a seperate menu or item page to change these things for a specific episode tho. That's why I made another concept of how I could see these selectors work or you could say I made a slight redesign of the item page with additions of selectors.

For the episodes you can see how long it is and when it would end so the description and title would move a little bit up. Then I would give a smaller selector that shows what preferences have been selected. Then you can click on that smaller selector that would give just icons so it wouldn't take that much space and it still gives all the options available for that episode. This selector is seen at the bottom. When an option in the drop down menu of an episode gets  changed, it will try to use the same option for all the other  episodes of that serie if that option is available. Thanks to Tiwenty for this idea :) This way it is really fast and easy to change settings for a whole series without the need for an extra menu as proposed in the previous design. This drop down menu was to include some sort of dial menu proposed by Timscampi. I would just make it smaller. (The icons were just temporary, but those I had easy access to) 

I also showed how far you are with the episode with a blue bar for now. As that is used at multiple parts in the design. (I guess this was already the plan, just that it didn't report it back yet.)

I tried to keep this design in-line with the movie item page and I think it still looks pretty clean. I was going for a one-page design that would be easy to read and you wouldn't need another page to select the rest. But for some odd undefined or missing audio/subtitle tracks, there would still be an option to change this yourself before going into the episode. 

![Improved Series Item Page](https://user-images.githubusercontent.com/42120256/103876742-55efcd80-50d4-11eb-8395-1a7c339acbab.png)

I guess that was it ;) Hope you like it

**Comment by Bitwolfies** [comment]
Absolutely love it

**Comment by GlassedSilver** [comment]
Great taste in series and a very lovely design concept.

Could one integrate setting defaults for the entire show with a similar design as well?
Either way, looks incredible!
